services/backend/app/crud/crud_stats.py

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Dict, List, Optional, Union
+
+from sqlalchemy.orm import Session, Query
+from sqlalchemy.sql.functions import func
+
+from app.crud.base import CRUDBase
+from app.models.stats import Stats
+
+class CRUDStats(CRUDBase[Stats]):
+
+    def get_
```

Comment:
Better to comment this and add `pass` after class declaration

Reply:
done allready